packages/react/src/components/ImpactBadge/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { forwardRef } from 'react';
+
+import Badge, { BadgeLabel, BadgeProps } from '../Badge';
```

Comment:
Minor nit:

Instead of using the type export, we should use `React.ComponentProps<typeof BadgeLabel>` here. This gives us greater flexibility in not having to be concerned with having to export the prop type.